res/layout/rate_me_dialog.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ownCloud Android client application
+
+  Copyright (C) 2016 ownCloud GmbH.
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License version 2,
+  as published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<LinearLayout
```

Comment:
The dialog design is pretty good but I would align the content to the left, since default Android UI tends to have the headers and text in that position while iOS usually show the content more centered in the middle, like you have done in your dialog. I think something like the next dialog would be better

<img src="https://user-images.githubusercontent.com/15261363/35974969-6a5c7cea-0cdb-11e8-80e1-4b7062e23ca7.png" width="50%">